src/main.ts

```diff
@@ -9,17 +9,17 @@ const fetchRPCAddr = async (netconfigURL: string) => {
   return rpcAddrs[Math.floor(Math.random() * rpcAddrs.length)];
 };
 
-const networkConfigUrl = (() => {
-  const agoricNetName =
-    new URLSearchParams(window.location.search).get('network') ?? 'main';
+const agoricNetName = new URLSearchParams(window.location.search).get('network') ?? 'main';
```

Comment:
Note: This is over 80 lines, but we don't have prettier setup in the repo so I should just do that. I've just been using the default vscode prettier.